modules/claims_api/spec/services/power_of_attorney_request_service/create_request_spec.rb

```diff
@@ -191,7 +191,15 @@
           response = subject.call
 
           expect(response['meta']).to include(expected_response['meta'])
-          expect(response.except('meta')).to match(expected_response.except('meta'))
```

Comment:
This assertion looks like it maybe wasn't intended to be removed, since it's the only line that tests that the non-meta parts of the response match the expected response.